packages/ui/src/components/va-avatar/VaAvatar.vue

```diff
@@ -13,6 +13,7 @@
       <img
         v-else-if="src"
         :src="src"
+        alt=""
```

Comment:
Maybe we should add here some example text, like a `Avatar`?

Reply:
It won't be useful for screenreaders (no information about image - just its purpose). Empty `alt` here is because without it screenreaders will read `src` (just ignoring with empty `alt`).

Reply:
Think i'll add `aria-hidden = true` to icon state and optional `alt` prop for image state (some others UI frameworks deal with it in such way).